backend/src/main/java/gov/cdc/usds/simplereport/db/repository/TestOrderRepository.java

```diff
@@ -15,33 +15,38 @@
 
 public interface TestOrderRepository extends AuditedEntityRepository<TestOrder> {
 
-	public static final String BASE_ORG_QUERY = "from #{#entityName} q "
-				+ "where q.organization = :org "
-				+ "and q.organization.isDeleted = false "
-				+ "and q.patient.isDeleted = false ";
-	public static final String FACILITY_QUERY = BASE_ORG_QUERY + " and q.facility = :facility ";
-	public static final String IS_PENDING = " and q.orderStatus = 'PENDING' "; 
-	public static final String IS_COMPLETED = " and q.orderStatus = 'COMPLETED' ";
+    public static final String BASE_ORG_QUERY = "from #{#entityName} q "
+                + "where q.organization = :org "
+                + "and q.organization.isDeleted = false "
+                + "and q.patient.isDeleted = false ";
+    public static final String FACILITY_QUERY = BASE_ORG_QUERY + " and q.facility = :facility ";
+    public static final String IS_PENDING = " and q.orderStatus = 'PENDING' "; 
+    public static final String IS_COMPLETED = " and q.orderStatus = 'COMPLETED' ";
+    public static final String HAS_EVENT = " and q.testEvent IS NOT NULL ";
```

Comment:
How would this be false if the status is COMPLETED? I mean, I guess the answer is that we could have corrupted our data, but that's a thing that seems worth checking somewhere. 😕